patches/xctestrunner_module_name.patch

```diff
@@ -0,0 +1,147 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
For xctestrunner we can ping the core maintainer with the patch - did you make a PR for that? If they are MIA we can push to a rules_ios fork if it comes to that.